docs/BUILD.bazel

```diff
@@ -106,7 +106,7 @@ genrule(
         cp -rL docs/source source
 
         mkdir -p -- source/error-codes/self-service
-        cp -- $(location //docs:generate-docs-error-code-inventory-into-rst-file) source/app-dev/grpc/error_codes_inventory.rst
+        cp -L -- $(location //docs:generate-docs-error-code-inventory-into-rst-file) source/app-dev/grpc/error-codes-inventory.rst
```

Comment:
`-L` made no difference on my system but it's used consistently below so adding it here as well